schedule.md

```diff
@@ -0,0 +1,14 @@
+## Time Schedule

```

Comment:
I don't think we should have the same time schedule in two separate places. keep it all on the main website so that if we need to update something, we can do it quickly at once.

Reply:
I removed the schedule page